QtOffscreenViews/QAndroidOffscreenWebView.cpp

```diff
@@ -397,6 +397,14 @@ void QAndroidOffscreenWebView::goForward()
 	}
 }
 
+void QAndroidOffscreenWebView::setWebContentsDebuggingEnabled(bool enabled)
+{
+    if (QJniObject * view = offscreenView())
+    {
+        view->callVoid("setWebContentsDebuggingEnabled",(jboolean)enabled);
```

Comment:
`view->callVoid("setWebContentsDebuggingEnabled", static_cast<jboolean>(enabled)); `

Reply:
The C-style cast now causes a warning in current compilers.

Reply:
Yes, this is fixed on my last commit. Please review.